config/application.yml.default

```diff
@@ -581,6 +581,7 @@ test:
   max_mail_events: 2
   otp_delivery_blocklist_findtime: 1
   otp_delivery_blocklist_maxretry: 2
+
```

Comment:
Any reason for this break?

Reply:
ah, I had that change staged, but not committed. Fixed that in [d261989](https://github.com/18F/identity-idp/pull/11899/commits/d26198960fcab8230268f915057c7e8fc56aed7e)